packages/web-components/src/components/locale-modal/locale-modal-composite.ts

```diff
@@ -181,32 +192,50 @@ class DDSLocaleModalComposite extends HybridRenderMixin(LitElement) {
                 ?invalid="${countryList.length === 0 ||
                 massagedCountryList?.find(({ region }) => region === name) ===
                   undefined}"
-                name="${name}"></dds-region-item>
+                name="${name}"
+                @click="${this._handleRegionClick}"></dds-region-item>
             `;
           })}
         </dds-regions>
-        <dds-locale-search
-          close-button-assistive-text="${ifNonNull(searchClearText)}"
-          label-text="${ifNonNull(searchLabel)}"
-          placeholder="${ifNonNull(searchPlaceholder)}"
-          availability-label-text="${ifNonNull(availabilityText)}"
-          unavailability-label-text="${ifNonNull(unavailabilityText)}">
-          ${massagedCountryList?.map(
-            ({ country, href, language, locale, region }) => html`
-              <dds-locale-item
-                country="${country}"
-                href="${href}"
-                language="${language}"
-                locale="${locale}"
-                region="${region}">
-              </dds-locale-item>
+
+        ${this.chosenRegionRender
```

Comment:
You might be able to get away with checking for truthiness of `this.chosenRegion` here, at which point you'd no longer need `this.chosenRegionRender` at all.

Reply:
i originally tried this and for some reason couldn't get it to work, hence the new var. but i will try again! thank you